test/src/xquery/namespaces.xql

```diff
@@ -272,3 +272,15 @@ declare
 function nt:qname-lhs-compare() {
     xs:QName('test') eq 'test'
 };
+
```

Comment:
Erm I think these tests might be in the wrong file. They don't seem to be namespace related tests ;-)

Reply:
I actually put them in this file because.... you put in a similar test here recently...

Reply:
Oh! Doesn't mean I got it right though ;-)

Reply:
Ah okay, so my tests were around `QNames`. Maybe could have gone somewhere else, they have a tenuous relationship to namespaces ;-)

Reply:
I saw the compare thingy.... :-)